sys/net/crosslayer/ng_netif/ng_netif_addr_from_str.c

```diff
@@ -14,47 +15,51 @@
 
 #include "net/ng_netif.h"
 
-static uint8_t _parse_byte(char *str)
+static int _dehex(char c)
```

Comment:
`inline` is only a suggestion and we cannot tell for every platform if inlining is actually better or worse. The compiler will inline the function on its own if it thinks that optimizes something.